app/model/ApplicationStatus.scala

```diff
@@ -24,6 +24,8 @@ object ApplicationStatus extends Enumeration {
   val WITHDRAWN, CREATED, IN_PROGRESS, SUBMITTED = Value
   val PHASE1_TESTS = Value
 
+  implicit def toString(applicationStatus: ApplicationStatus): String = applicationStatus.toString
```

Comment:
I don't think you need this, enum's have a default toString - which you're using :-)